tests/Transports.AspNetCore.Tests/Transports.AspNetCore.Tests.csproj

```diff
@@ -11,6 +11,10 @@
     <TargetFrameworks>net6</TargetFrameworks>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <NoWarn>$(NoWarn);CA1835</NoWarn>
```

Comment:
CA1835 should be a global ignore within tests, as some frameworks tested do not support such a change.  Within the main project we use ifdef to use the ideal overload.

Reply:
ok